.github/workflows/PR-3.4.yaml

```diff
@@ -2,9 +2,14 @@ name: PR:3.4
 
 on:
   pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
```

Comment:
> labeled
> unlabeled

Why do we need this again?

Reply:
We discussed with @asmorkalov and decided to re-trigger pipelines if there are any updated labels. Your point was about triggering a lot of pipelines in this case. But another change of this PR is about cancelling old jobs what are in progress and run only the latest.

Reply:
I think it will be easier than to add label and then manually push the latest commit for a PR again

Reply:
> again

We already discussed this previously (check internal meeting minutes from 2022-04-29).

Before "Merge" I usually put extra labels (up to 5) or remove stalled labels like "needs rebase" or "incomplete".
It is annoying that CI is re-triggered during that (need to wait for 1+ extra hour to pass CI from scratch).

Also setting of "incomplete" label on PR should not trigger CI. Because nobody needs these results.

Reply:
I got your point about several labels at the time before merging. In this case to re-trigger jobs on a label certainly isn't good way.

Discussed on a local meeting: try to add a whitelist of labels which should trigger CI.